tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/PackageCheckType.cs

```diff
@@ -0,0 +1,22 @@
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Defines the types of validation checks that can be run on SDK packages.
+/// </summary>
+public enum PackageCheckType
```

Comment:
```suggestion
public enum PackageCheckName
```

Nit, maybe `PackageCheckName` would be a little clearer? At first glance, this name made me think that each enum value represented a category of checks and that there would be multiple checks of each type. Might just be me though